core-bundle/src/Resources/contao/forms/FormTextArea.php

```diff
@@ -140,7 +142,11 @@ public function __get($strKey)
 				break;
 
 			case 'value':
-				return \StringUtil::specialchars(str_replace('\n', "\n", $this->varValue), false, true);
+				return \StringUtil::specialchars(str_replace('\n', "\n", $this->varValue));
+				break;
+
+			case 'rawValue':
+				return str_replace('\n', "\n", $this->varValue);
```

Comment:
Is there a special reason why we replace `'\n'` with an actual line break?
IMO we should just return `$this->varValue` here.

Reply:
Changed in 69e587047f61ab6bc0e45e6ac3717a74d4b3ad77.